cmd/example/register-cluster_complicated-exemple.json

```diff
@@ -0,0 +1,280 @@
+{
+  "runtimeInput": {
+    "name": "ad0d842",
+    "description": "",
+    "labels": {
+      "scenarios": [
+        "DEFAULT"
+      ]
+    }
+  },
+  "kymaConfig": {
+    "version": "1.19.0",
+    "profile": "Production",
+    "conflictStrategy": "Replace",
```

Comment:
do we need conflict strategy? I think we will drop such logic.

Reply:
Done